Error running npm run dev — Cannot find module '../lightningcss.linux-x64-gnu.node'

### 🐛 Bug Report: Error running `npm run dev` — Cannot find module '../lightningcss.linux-x64-gnu.node'

#### Description

After following the setup steps from the README, I encountered an error when running the development server.  
The error message indicates that the module `../lightningcss.linux-x64-gnu.node` cannot be found.  
All environment variables have been correctly filled in according to the `.env.example` file.

#### Steps to Reproduce

1. Clone the repository:

   ```bash
   git clone https://github.com/openai/openai-chatkit-starter-app
   cd openai-chatkit-starter-app
   ```

2. Install dependencies:

   ```bash
   npm install
   ```

3. Fill in all required environment variables following the `.env.example` template.

4. Run the development server:

   ```bash
   npm run dev
   ```

5. The following error appears in the terminal:

   ```
   Build Error
   
   Cannot find module '../lightningcss.linux-x64-gnu.node'
   
   Error: Cannot find module '../lightningcss.linux-x64-gnu.node'
   Require stack:
   - /path/to/openai-chatkit-starter-app/node_modules/lightningcss/node/index.js
   - /path/to/openai-chatkit-starter-app/node_modules/@tailwindcss/node/dist/index.js
   - ...
   ```

#### Environment

- **Operating System:** Linux (x64)
- **Node.js version:** e.g. v22.20.0
- **Next.js version:** 15.5.4 (as shown in the console)  
- **Expected behavior:** The development server should start without build errors.


Possible reasons include:

- The correct binary for the current system wasn’t downloaded.
- Compatibility issues with certain Linux distributions.
- The package might need to be rebuilt manually using `npm rebuild lightningcss`.

#### Screenshot

![Error Screenshot](https://github.com/user-attachments/assets/d1bc50cc-21d2-4839-93e0-b5a3460ae231)

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -25,4 +25,7 @@ jobs:
 
       - name: Run ESLint
         run: npm run lint
+
+      - name: Run build
+        run: npm run build
   

--- a/package.json
+++ b/package.json
@@ -14,6 +14,10 @@
     "react": "^19.2.0",
     "react-dom": "^19.2.0"
   },
+  "optionalDependencies": {
+    "lightningcss-linux-x64-gnu": "1.30.1",
+    "@tailwindcss/oxide-linux-x64-gnu": "4.1.11"
+  },
   "devDependencies": {
     "@eslint/eslintrc": "^3",
     "@tailwindcss/postcss": "^4",

